src/btf.cpp

```diff
@@ -72,6 +72,16 @@ BTF::~BTF()
 
 void BTF::load_kernel_btfs(const std::set<std::string> &modules)
 {
+  vmlinux_btf = btf__load_vmlinux_btf();
+  if (libbpf_get_error(vmlinux_btf))
+  {
+    if (bt_debug != DebugLevel::kNone)
+      LOG(ERROR) << "BTF: failed to find BTF data for vmlinux";
```

Comment:
Also log the errno from `libbpf_get_error()` here?

Reply:
Done.